### DISCUSSION TITLE
'unknown word' error when using less preprocessor

### DISCUSSION THREAD

**Discussion by minnieoo** [openstyles/stylus#1716] [General]
i have 'unknown word' error when using less preprocessor. i am user userCss option. the mixin refuses to work, but using these same settings with less & mixins works fine on OTHER styles, i even tried copying other styles where it works, and just replacing the inner css and still this error. here is a link to the user style so you can take a look: https://userstyles.world/style/13516/rainbowbin-not-working

![image](https://github.com/openstyles/stylus/assets/137783738/badea27c-3638-4a56-baa7-4bdb6b20db26)

**Comment by tophf** [comment]
Our linters can't check some of the advanced preprocessor features yet.
For now, you can collapse the Issues panel or disable `CSS Linter` option to hide the error.

**Reply by tophf** [reply]
You can also use an external editor locally and auto-track the edits using [live reload](https://github.com/openstyles/stylus/wiki/Writing-UserCSS#initial-installation-and-live-reload).

**Reply by minnieoo** [reply]
i'm confused, it works fine with no errors on other styles though

**Reply by tophf** [reply]
I guess those styles don't use this particular feature.

**Reply by minnieoo** [reply]
but they do use mixins

**Reply by tophf** [reply]
I can't guess further without seeing the actual code.

**Reply by tophf** [reply]
BTW Your current style can't be parsed by `LESS` because it doesn't support multiple selectors inside `:is`, it's a bug in the preprocessor as seen in https://lesscss.org/less-preview/, so you'll have to use a workaround e.g. escape the comma or something else.
